### PR TITLE
docs: fix og-image

### PR DESCRIPTION
**Pull request by felangel**
## Description

- docs: fix og-image

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: site/docusaurus.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const config = {
   title: 'Very Good CLI',
   tagline:
     'A Command-Line Interface to generate scalable templates and use helpful commands.',
-  url: 'https://cli.vgv.dev',
+  url: 'https://verygoodopensource.github.io',
```
Comment (wolfenrain) [inline site/docusaurus.config.js:12]: Intentional?

**Review by wolfenrain — APPROVED** [review]

File: site/docusaurus.config.js (hunk 1 above)
Comment (scarletteliza) [reply site/docusaurus.config.js:12]: yeah we don't have a custom domain yet...

Comment (scarletteliza) [reply site/docusaurus.config.js:12]: we'll revert when we do

Comment (scarletteliza) [reply site/docusaurus.config.js:12]: we'll revert when we do

Comment (felangel) [reply site/docusaurus.config.js:12]: yeah in order for the og images to work. we'll need to revert this when we configure the subdomain.

**Pull request merged**